networking_ccloud/tests/unit/extensions/test_extensions.py

```diff
@@ -215,13 +225,17 @@ def test_network_ensure_interconnects(self):
             self._make_segment(network_id=network_id, network_type='vxlan',
                                segmentation_id=424242,
                                tenant_id="test-tenant", fmt='json')['segment']
+            objs = self.ctx.session.query(segment_models.NetworkSegment).filter_by(physical_network=None,
+                                                                                   network_type='vxlan')
+            objs.update({'segment_index': 0})
```

Comment:
I don't get this part. Is it because you cannot specify the `segment_index` in `self._make_segment()`?

Reply:
```
{"NeutronError": {"type": "HTTPBadRequest", "message": "Unrecognized attribute(s) 'segment_index'", "detail": ""}}
```
`_make_segment()` doesn't accept this as an argument. If there is a way to do that I haven't looked hard enough yet.

Reply:
A comment to explain this, would have been nice, but not required.